### PR TITLE
Add support to run dash cases hero using Hardware traffic generator in sonic-mgmt container.

### PR DESCRIPTION
**Pull request by vikumarks**
* Add python packages for DASH testcases.

dpugen==0.0.5
macaddress
munch
ctypesgen
pandas
PyYAML
ixload

* Add python packages for DASH testcases.

removed munch and macaddress, as it comes with dpugen

#### Why I did it
As part of the dash cases integration in sonic-mgmt container .
#### How I did it
Added required python packages dependency in docker file.
#### How to verify it
Run dash script inside sonic-mgmt container.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 1]
```diff
@@ -87,11 +87,16 @@ RUN pip install allure-pytest==2.8.22 \
...
+                ixload \
                 statistics \
                 tabulate \
                 textfsm==1.1.3 \
                 thrift==0.11.0 \
                 virtualenv \
+                dpugen==0.0.5 \
+                ctypesgen \
+                pandas \
+                PyYAML \
```
Comment (wangxin) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:99]: Can you add the dependent packages to python3 environment too? We are in the progress of migrating to python3 and deprecating python2 environment.
BTW, can you add the packages in alphabetic order?

**Comment by theasianpianist** [comment]
I think I'm a little confused, why do we need to add these packages? I've been able to run DASH test cases using the existing sonic-mgmt container.

File: dockers/docker-sonic-mgmt/Dockerfile.j2 (hunk 1 above)
Comment (vikumarks) [reply dockers/docker-sonic-mgmt/Dockerfile.j2:107]: Added for python3 also

**Comment by vikumarks** [comment]
> I think I'm a little confused, why do we need to add these packages? I've been able to run DASH test cases using the existing sonic-mgmt container.

This is needed to run HERO Test using Hardware traffic generator.
I have updated the PR description to make it more clear.

**Comment by vkuma82** [comment]
@theasianpianist / @wangxin Can one of you please see what else i need to do to make this PR accepted and merged?

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 2]
```diff
@@ -54,11 +54,14 @@ RUN pip install allure-pytest==2.8.22 \
                 cffi==1.12.0 \
                 contextlib2==0.6.0.post1 \
                 cryptography==3.3.2 \
+                ctypesgen \
```
Comment (wangxin) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:57]: Can you move it up one line to ensure alphabetical order?

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 3]
```diff
@@ -100,6 +104,7 @@ RUN pip install allure-pytest==2.8.22 \
                 textfsm==1.1.3 \
                 thrift==0.11.0 \
                 virtualenv \
+                PyYAML \
```
Comment (wangxin) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:107]: Can you move up this line to ensure alphabetical order?

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 4]
```diff
@@ -198,11 +203,14 @@ RUN python3 -m pip install aiohttp \
                            cffi \
                            contextlib2==0.6.0.post1 \
                            cryptography==3.3.2 \
+                           ctypesgen \
```
Comment (wangxin) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:206]: Can you move up this line to ensure alphabetical order?

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 5]
```diff
@@ -247,7 +256,8 @@ RUN python3 -m pip install aiohttp \
                            tabulate \
                            textfsm==1.1.2 \
                            thrift==0.11.0 \
-                           virtualenv
+                           virtualenv \
+                           PyYAML
```
Comment (wangxin) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:260]: Can you move up this line to ensure alphabetical order?

**Pull request closed without merging**